ga4gh/frontend.py

```diff
@@ -25,6 +26,7 @@
 
 
 app = flask.Flask(__name__)
+urls = []
```

Comment:
No so sure about this. Do we _really_ need a global variable to do this? If we do, then I would all it `_urls` to ensure that it's module private.